yak/CMakeLists.txt

```diff
@@ -82,6 +82,7 @@ target_link_libraries(${PROJECT_NAME}
   ${CUDA_CUDA_LIBRARY}
   ${PCL_LIBRARIES}  # BUG: shouldn't need this, but get -fPIC errors if it's not included
   Eigen3::Eigen)
+set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS -fPIC)
```

Comment:
Would it be clearer to write this using a named CMake property vs. the raw flag?
```
 set_target_properties(${PROJECT_NAME} PROPERTIES POSITION_INDEPENDENT_CODE ON) 
```

Reply:
Sure, I did realize that was an option as well. I will update the commit.